Sources/MockService.swift

```diff
@@ -35,39 +35,6 @@ open class MockService: NSObject {
     self.errorReporter = errorReporter
   }
 
-  ///
-  /// Convenience Initializer
-  ///
-  /// - parameter provider: Name of your provider (eg: Calculator API)
-  /// - parameter consumer: Name of your consumer (eg: Calculator.app)
-  /// - parameter pactVerificationService: Your customised `PactVerificationService`
-  ///
-  /// Use this initialiser to use the default XCodeErrorReporter
-  ///
-  @objc(initWithProvider: consumer: andVerificationService:)
```

Comment:
The initializer sets `XCodeErrorReporter` as its error reporter, and that is dependent on Nimble
I can add it back, but then it would default it to `XCTestErrorReporter`, if that's ok

Reply:
@rubencagnie I think I missed something in the first review. How is the `test` reference in XCTestErrorReporter meant to be assigned? 
I would have liked XCTestErrorReporter to be the default, but I don't really understand how it will get a reference to the underlying XCTestCase class....?

Reply:
We could require the initializer to pass in the XCTest?